lib/node_modules/@stdlib/math/base/special/besselj1/README.md

```diff
@@ -0,0 +1,71 @@
+# j1
+
+> Compute the [Bessel function of the first kind][bessel-j] of order one.
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
```

Comment:
Heads up: due to how a dependency (`remark`) generates Markdown, our Markdown style has updated a bit. Notably, a space no longer exists between the fence and the language. To avoid a needless diff in future builds, you can remove the space ahead-of-time and spare file churn.

cc @Planeshifter